yara/yara3/Makefile.acr

```diff
@@ -41,7 +41,7 @@ install:
 	${INSTALL_LIB} ${MOD}.${LIBEXT} $(R2PM_PLUGDIR)/${MOD}.${LIBEXT}
 	rm -rf $(R2PM_PLUGDIR)/yara
 	mkdir -p $(R2PM_PLUGDIR)/yara
-	cp -f ../rules-yara3/* $(R2PM_PLUGDIR)/rules-yara3
+	cp -rf ../rules-yara3/ $(R2PM_PLUGDIR)/
```

Comment:
if rules-yara3 directory exists in target it will copy it inside the other one. which can be non-desired behaviour